package.json

```diff
@@ -132,13 +133,64 @@
 					"description": "Note: We only send a few very basic events and the extension version number/platform :-)"
 				}
 			}
+		},
+		"debuggers": [{
+			"type": "dart-cli",
+			"label": "Dart command line",
+			"enableBreakpointsFor": {
```

Comment:
If this is just Run for now (without debug, and assuming we might ship this before Debug is done) would it be better to comment this out for now - I think people might get confusing if they can add breakpoints and run in a way that it looks like the debugger is attached, but they don't get hit?